src/Context/Cart.tsx

```diff
@@ -1,30 +1,33 @@
 import { createContext, useContext, useReducer, useEffect, ReactNode } from "react";
 import { Vinyl } from "../utils/types";
 
-// Define the shape of the cart state
+export const ADD_TO_CART = "ADD_TO_CART";
+const DECREASE_QUANTITY = "DECREASE_QUANTITY";
+
+
 interface CartState {
   cartItems: Vinyl[];
   cartCount: number;
   totalPrice: number;
 }
 
-// Define the shape of the actions
+
 interface CartAction {
-  type: "ADD_TO_CART" | "REMOVE_FROM_CART" | "DECREASE_QUANTITY";
+  type: typeof ADD_TO_CART | typeof DECREASE_QUANTITY;
   payload: Vinyl;
 }
```

Comment:
Could this interface be moved to the types fil. And then you can import it similar to how you have done with the vinyl types

import { CartAction } from "../utils/types";

Reply:
Thank Josh. I moved it to the type.ts. By doing this I have to twik some costants.